edm4hep.yaml

```diff
@@ -594,3 +594,20 @@ datatypes:
       - edm4hep::HitLevelData hitData   //hit level data
     OneToOneRelations:
       - edm4hep::Track track           //the corresponding track.
+
+interfaces:
+  edm4hep::TrackerHit:
+    Description: "Tracker hit interface class"
+    Author: "Thomas Madlener, DESY"
+    Members:
+      - uint64_t cellID // ID of the sensor that created this hit
+      - int32_t type // type of the raw data hit
+      - int32_t quality // quality bit flag of the hit
+      - float time [ns] // time of the hit
+      - float eDep [GeV] // energy deposited on the hit
+      - float eDepError [GeV] // error measured on eDep
+      - edm4hep::Vector3d position [mm] // hit position
+      - std::array<float, 6> covMatrix // covariance of the position (x,y,z) stored as lower triangle matrix, i.e. cov(x,x), cov(y, x), cov(y, y), cov(z, x), cov(z, y), cov(z, z)
```

Comment:
```suggestion
```

Remove the 3D covariance matrix as discussed during the last EDM4hep meeting (Jan 23).

Reply:
Done